File: WS Cube Tech/LogCatExample/app/src/main/java/com/example/logcatexample/MainActivity.java
package com.example.logcatexample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Log.d("Test","Test log");

        String TAG = "First Log";
        Log.d(TAG,"My first log message");
    }
}

File: WS Cube Tech - Kotlin/IntentPassingKotlin/app/src/main/java/com/example/intentpassingkotlin/MainActivity.kt
package com.example.intentpassingkotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.intentpassingkotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Inflate the layout using ViewBinding
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        // Set click listener for the "Next" button
        binding.btnNext.setOnClickListener {
            // Create an Intent to navigate to the SecondActivity
            val iNext = Intent(this@MainActivity, SecondActivity::class.java)

            // Start the SecondActivity
            startActivity(iNext)
        }
    }
}
